taipy/gui/hook.py

```diff
@@ -0,0 +1,40 @@
+import typing as t
+
+from taipy.logger._taipy_logger import _TaipyLogger
+
+from .utils.singleton import _Singleton
+
+
+class Hook:
+    method_names: t.List[str] = []
```

Comment:
Is this line useful?

Reply:
it is, very. As a sense of base class implementation